addons/website_forum/static/src/js/website_forum.js

```diff
@@ -357,42 +368,19 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
                     sticky: false,
                 });
             } else {
-                $acceptAnswerLinks.removeClass('oe_answer_true')
-                                  .addClass('oe_answer_false');
-                $link.toggleClass('oe_answer_true', !!data)
-                     .toggleClass('oe_answer_false', !data);
-
-                // TODO in master, review the utility of this function...
-                self._onCheckAnswerStatus(ev);
+                _.each(self.$('.forum_answer'), function (answer) {
+                    var isCorrect = $(answer).is(target) && data;
+                    var $toggler = $(answer).find('.o_wforum_validate_toggler');
+                    var newHelper = isCorrect ? $toggler.data('helper-decline') : $toggler.data('helper-accept');
 
-                // If we are removing an accepted answer, reload the page as the
-                // design is quite different with or without an accepted answer.
-                if ($link.closest('.validate_answer').length) {
-                    window.location.reload();
-                }
+                    $(answer).toggleClass('o_wforum_answer_correct', isCorrect);
+                    $toggler.tooltip('dispose')
+                            .attr('data-original-title', newHelper)
+                            .tooltip({delay: 0});
```

Comment:
Note to myself: maybe instead of destroying and recreating tooltip, we should initialize tooltip with a "title" function to fet the correct one according to the class.